models/endoscopic_tool_segmentation/configs/metadata.json

```diff
@@ -1,7 +1,8 @@
 {
     "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
-    "version": "0.4.1",
+    "version": "0.4.2",
     "changelog": {
+        "0.4.2": "update the readme file with TensorRT convert",
         "0.4.1": "add name tag",
         "0.4.0": "add support for multi-GPU training and evaluation",
         "0.3.2": "restructure readme to match updated template",
```

Comment:
Mark: may need to modify `monai_version` after 1.2 is released.

Reply:
Hi @yiheng-wang-nv ,

According to our plan, please create a `monai_1.2` branch in this repo ASAP, and let's merge this PR to that branch first.
@binliunls Please update the MONAI version to 1.2.0 directly in this PR.

Thanks.

Reply:
@yiheng-wang-nv I see you already created `dev-1.2` branch, @binliunls let's use it.

Thanks.